aesd-char-driver/aesd-circular-buffer.c

```diff
@@ -1,6 +1,6 @@
 /**
  * @file aesd-circular-buffer.c
- * @brief Functions and data related to a circular buffer implementation
+ * @brief Functions and data related to a circular buffer imlementation
```

Comment:
@Rajs060699 curious why you are re-adding a typo here which was fixed in a previous semester.  Can you explain this?

Reply:
This repo bears a striking resemblance to https://github.com/cu-ecen-aeld/assignments-3-and-later-shreyan-collab

Especially before commit https://github.com/cu-ecen-aeld/assignments-3-and-later-Rajs060699/commit/7f70e8fb63d50b24197f9261b0c930fb5d2fe27e which appears to just change variable names and removes comments.

Are you doing your own work?

Reply:
Yes Sir,Im doing my own work